class-astra-theme-customizer-reset.php

```diff
@@ -125,22 +125,29 @@ public function ajax_customizer_reset() {
 		public function controls_scripts() {
 
 			// Enqueue JS.
-			wp_enqueue_script( 'astra-theme-customizer-reset', ASTRA_THEME_CUSTOMIZER_RESET_URI . 'assets/js/customizer-reset.js', array( 'jquery', 'astra-customizer-controls-toggle-js' ), null, true );
+			wp_enqueue_script( 'astra-theme-customizer-reset', ASTRA_THEME_CUSTOMIZER_RESET_URI . 'assets/js/customizer-reset.js', array( 'jquery', 'astra-customizer-controls-toggle-js' ), true, true );
```

Comment:
@shubhw12 Please add version number instead of `true`

Reply:
Done